04-spatial-operations.Rmd

```diff
@@ -140,13 +148,14 @@ For further discussion of this decision, and some controversy, see the (closed)
 
 But what is `sel`?
 It is not, as one might imagine, a `logical` vector.
+<!-- What do you mean by two-dimensional? One can misunderstand this and believe that you are talking about 2 columns. Or are you already referring to sel_matrix? --> 
```

Comment:
What do you mean by two-dimensional? One can misunderstand this and believe that you are talking about 2 columns. Or are you already referring to `sel_matrix`?

Reply:
Solved now - thanks for the comment, there were defo issues here.